common/djangoapps/third_party_auth/migrations/0004_lti_tool_consumers.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=C,E,F,R,W
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'LTIProviderConfig'
+        db.create_table('third_party_auth_ltiproviderconfig', (
+            ('id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('change_date', self.gf('django.db.models.fields.DateTimeField')(auto_now_add=True, blank=True)),
+            ('changed_by', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['auth.User'], null=True, on_delete=models.PROTECT)),
+            ('enabled', self.gf('django.db.models.fields.BooleanField')(default=False)),
+            ('icon_class', self.gf('django.db.models.fields.CharField')(default='fa-signin', max_length=50)),
+            ('name', self.gf('django.db.models.fields.CharField')(max_length=50)),
+            ('lti_consumer_key', self.gf('django.db.models.fields.CharField')(max_length=255)),
+            ('lti_consumer_secret', self.gf('django.db.models.fields.CharField')(max_length=255)),
+            ('lti_max_timestamp_age', self.gf('django.db.models.fields.IntegerField')(default=10)),
```

Comment:
Also, we should probably hide fields like "secondary" from the django admin view, since they don't apply to this provider. (Or change the dependency tree so the LTI provider inherits from a simpler base class that doesn't include `secondary` or other irrelevant fields)